src/pages/NewResearcherConsole.js

```diff
@@ -1,73 +1,95 @@
-import { useState, useEffect, useRef, useCallback, useMemo } from 'react';
+import { useState, useEffect, useRef, useCallback } from 'react';
 import { div, h, img } from 'react-hyperscript-helpers';
-import {cloneDeep, map, findIndex, isEmpty, pullAt, get, flow, keys, isNil, head, concat} from 'lodash/fp';
-import TabControl from '../components/TabControl';
+import {cloneDeep, map, findIndex, isEmpty, get, flow, keys, isNil, head, concat, replace} from 'lodash/fp';
 import { Styles } from '../libs/theme';
 import { Collections, DAR } from '../libs/ajax';
 import { DarCollectionTableColumnOptions, DarCollectionTable } from '../components/dar_collection_table/DarCollectionTable';
 import accessIcon from '../images/icon_access.png';
 import { Notifications, searchOnFilteredList, getSearchFilterFunctions } from '../libs/utils';
 import SearchBar from '../components/SearchBar';
-import DarDraftTable from '../components/dar_drafts_table/DarDraftTable';
 
-//helper function with a built in delay to allow skeleton loader to show when data is loading or when the user switch tabs
-//primarily done to make the tab switching more obvious
-const delayedLoadingUpdate = async ({setIsLoading}) => {
-  return await Promise.resolve(
-    setTimeout(() => setIsLoading(false), [400])
-  );
+const createPropertiesForDraft = (keys, values) =>
+  keys.map((propertyKey, index) => ({
+    propertyKey,
+    propertyValue: values[index],
+  }));
+
+const formatDraft = (draft) => {
+  const { data, referenceId, id } = draft;
+  const {
+    partialDarCode,
+    projectTitle,
+    datasets,
+    isThePi,
+    piName,
+    institution,
+    investigator,
+  } = data;
+
+  const keys = ['isThePi', 'piName'];
+  const values = [isThePi, piName];
+  const output =  {
+    darCode: replace('temp', 'DRAFT')(partialDarCode),
+    referenceId,
+    darCollectionId: id,
+    projectTitle,
+    isDraft: true,
+    createDate: 'Unsubmitted',
+    datasets,
+    institution,
+    createUser: {
+      displayName: investigator,
+      properties: createPropertiesForDraft(keys, values),
+    },
+  };
+  return output;
+};
+
+const filterFn = getSearchFilterFunctions().darCollections;
+
+const redirectByReferenceId = (darCollection, history) => {
+  const referenceId = darCollection.referenceId || flow(
+    get('dars'),
+    keys,
+    head
+  )(darCollection);
```

Comment:
Since a submitted collection splits off a dar for each dataset, the `/dar_application/${referenceId}` page only displays a single dataset name. See https://github.com/DataBiosphere/duos-ui/pull/1590 for new read-only view that displays the names of all datasets in the collection. Drafts can still use this path